Makefile

```diff
@@ -33,11 +32,15 @@ endif
 
 ##@ Dependencies
 .PHONY: install-all-deps
-install-all-deps: install-build-deps install-site-deps ## Install all dependencies for building both binary and user docs)
+install-all-deps: install-site-deps ## Install all dependencies for building both binary and user docs)
```

Comment:
why is install-all-deps no longer installing the other dependencies?

Reply:
this looks like a mistake - will re-add

Reply:
corrected